README.md

```diff
@@ -59,12 +58,13 @@ BUILD_DIR=</path/to/llvm/build> \
 
 Optional flags:
 ```
-TEST_TARGET="X86_64;ARM" LNT_BIN=</path/to/lnt>
+TEST_TARGET="X86_64;ARM"
+LNT_BIN=</path/to/lnt> // if the name and location of the virtual environment is not ~/mysandbox.
```

Comment:
We might want to say: `If this is not specified then LNT is picked up from the default location ~/mysandbox/bin/lnt`.